imap_processing/codice/decom.py

```diff
@@ -0,0 +1,22 @@
+""" This is an example of how to use the space_packet_parser module to parse with a XTCE file.
```

Comment:
Is this file only ever going to be an example? If yes, then I would move it out of the `imap_processing/codice` folder and into some kind of example folder. Actually, anything that is more general use and NOT specifically codice related should go into a separate subfolder. Maybe `packet_utils`? I don't love utils folders in general because they're so vague. But in this case, it genuinely is a utility. Maybe ask about this in the slack channel and see if anyone has any strong opinions about where to put these XTCE generation files.